var/spack/repos/builtin/packages/lz4-java/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Lz4Java(Package):
+    """LZ4 compression for Java"""
+
+    homepage = "https://github.com/lz4/lz4-java"
+    url      = "https://github.com/lz4/lz4-java/archive/1.7.1.tar.gz"
+
+    version('1.7.1', sha256='2e5c4546788eddc76dd91008faa60f96059b216997a7e6009c97b0b2ba2b4ff7')
+
+    depends_on('ant', type='build')
+    depends_on('lz4', type='build')
+    depends_on('xxhash', type='build')
```

Comment:
If you are linking to these libraries/headers, these should be `type='link'`. If you also need them in `PATH` at build-time, they should be `type=('build', 'link')`.